doc/nrf/libraries.rst

```diff
@@ -31,6 +31,7 @@ Here you can find documentation for these libraries, including API documentation
    :glob:
 
    ../../include/*
+   ../../include/dfu/*
```

Comment:
How many libraries do we expect in here? Does it make sense to have a separate section for DFU libraries?

Reply:
For us we only have 1 but I don't know if someone else will add more. I don't think we need a separate section.